### DISCUSSION TITLE
Welcome to 20_python_project Discussions!

### DISCUSSION THREAD

**Discussion by israr96418** [israr96418/20_python_project#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by israr96418** [comment]
If you have any query, plz let me known

**Comment by ML-Engineer12** [comment]
Well done bro I like your code 
it soo much simple  and understandable
keep it up

**Comment by israr96418** [comment]
Thank you for your kind words
